samples/MediatR.Examples.Lamar/Program.cs

```diff
@@ -3,7 +3,7 @@
 using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
-using BlueMilk;
+using Lamar;
 using MediatR.Pipeline;
 
 namespace MediatR.Examples.BlueMilk
```

Comment:
Namespace?

Reply:
https://jeremydmiller.com/2018/03/05/renaming-bluemilk-to-lamar/

Reply:
Does that mean the namespace is still BlueMilk?

Reply:
No it's "Lamar"

Reply:
Derp. Replaced all the other names.